src/MUI.Web/wwwroot/app.css

```
@@ -32,6 +32,25 @@
+      screen from a Chinese, Japanese or Korean game is column art built on the assumption that a Han
+      glyph is exactly twice the advance of an ASCII one, which holds for Noto Sans Mono CJK, Sarasa
+      Mono and MS Gothic and does NOT hold for PingFang, Microsoft YaHei or Hiragino Sans — those are
+      proportional, and falling into one turns pkuxkx's bordered banner into a ragged staircase. The
+      Latin faces stay in front so an ASCII screen is unchanged.
+
+      No webfont, deliberately. Noto Sans Mono CJK SC is some eight megabytes a weight, and
+      subsetting is per-screen — a different glyph set for every game, regenerated on every crawl.
+      A reader with none of these installed gets their system's default CJK face, which is legible
+      and merely misaligned; that is the right end to lose at.
+    */
+    --mono-cjk: "Cascadia Mono", ui-monospace, "SFMono-Regular", Consolas,
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)